src/App.tsx

```diff
@@ -1,15 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { ITaskList } from '@components/types';
+import { v4 as uuidv4 } from 'uuid';
 import Title from './components/Title';
 import AddItemForm from './components/AddItemForm';
 import TaskDashboard from './components/TaskDashboard';
 
 function App() {
+  const [taskList, setListTask] = useState<ITaskList[]>([]);
```

Comment:
O padrão aqui seria seguir o mesmo nome do estado:

```suggestion
  const [taskList, setTaskList] = useState<ITaskList[]>([]);
```